smartlearner/update_rules/decreasing_learning_rate.py

```diff
@@ -24,17 +24,17 @@ def __init__(self, lr, dc=0.):
         self.lr = lr
         self.dc = dc
```

Comment:
Oh, wait, we already make sure `0 < dc < 1` (see the two lines before this one).